docker_multiarch/arm.crosscompile.android.mk

```diff
@@ -82,21 +82,6 @@ USE_OPENCV = 0
 # use openmp for parallelization
 USE_OPENMP = 1
 
-# MKL ML Library for Intel CPU/Xeon Phi
```

Comment:
@larroy

Reply:
Removing *MKL* flags from any arm cross compilation configs should be ok.  Thanks for updating.